codelists/forms.py

```diff
@@ -106,6 +107,14 @@ def clean_csv_data(self):
                 ]
             )
 
+        cleaned_header = [
+            '"' + h.strip('" ') + '"' if h[0] == '"' and h[-1] == '"' else h.strip()
```

Comment:
yeah, this looks better!

Reply:
although you'd still have a problem with escaped quotes in strings - does your source data contain those? `" my \"data\""`

Reply:
csvs are annoying

Reply:
I'm not sure what the reasonable expectation of user input is here. This isn't a particularly oft-used bit of functionality but we've had one instance already where some leading spaces in the header caused confusion and I thought I'd just put a quick fix in to stop it from happening again!

Reply:
This explicitly only removes the first and last chars of the string, which are the ones being tested  for presence of quotes. Notwithstanding a library to do this for us, I think this might be right?